src/Utils/Helpers/checkCurrencyFormat.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * @author Idan Gigi idangi@matrix.co.il
+ * @param {string} amount
+ * @returns {Boolean}
+ */
+export const checkCurrencyFormat = (amount) => {
```

Comment:
I think is better to put this file in clinikal-react/src/Utils/Helpers/validation